pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -939,17 +941,19 @@ func TestMakePodSpec(t *testing.T) {
 
 func TestMissingProbeError(t *testing.T) {
 	if _, err := MakeDeployment(revision("bar", "foo"), &logConfig, &traceConfig,
-		&network.Config{}, &obsConfig, &deploymentConfig); err == nil {
+		&network.Config{}, &obsConfig, &deploymentConfig, &asConfig); err == nil {
 		t.Error("expected error from MakeDeployment")
 	}
 }
 
 func TestMakeDeployment(t *testing.T) {
 	tests := []struct {
-		name string
-		rev  *v1.Revision
-		want *appsv1.Deployment
-		dc   deployment.Config
+		name    string
+		rev     *v1.Revision
+		want    *appsv1.Deployment
+		dc      deployment.Config
+		ac      *autoscalerconfig.Config
```

Comment:
```suggestion
		ac      autoscalerconfig.Config
```
see the usage of `dc`. But I guess you run into default 0/notAllow'd issue... May be just use a func instead? 🤷

Reply:
yep. i switched to using a func